aten/src/ATen/native/tags.yaml

```diff
@@ -42,6 +42,9 @@
   desc: |
           This tag indicates if an operator doesn't guarantee bitwise equivalence
           across different runs of an operator with identical inputs.
+- tag: needs_fixed_layout
```

Comment:
stride_order / stride_permutation seem good

Reply:
Got it, will change to `needs_fixed_stride_order`

Reply:
Note that the prior art is `empty_permute`, which uses `[0, 1, 2]` to indicate a contiguous tensor with dims permuted in some way.

By the way, if in reality, the operator takes a discontiguous tensor, what exactly is the semantics here?

Reply:
Currently, inductor will preserve the stride order/layout from the fx meta, but not the individual strides; so a noncontiguous tensor may have different stride values from eager mode, but will always permute those strides to the same layout that it was in in eager mode. 

@zou3519  and @eellison did talk about followups to potentially add another tag that would fix all strides and not just layouts, but concluded that it would require additional work tracking those.